scheduler/docs/scheduler-rest-api.asc

```diff
@@ -75,6 +75,7 @@ The request body is list of job maps with each entry containing a map of the fol
 |`max_runtime` | long | the maximum running time of the job in milliseconds. An instance that runs for more than max_runtime will be killed and job will be retried.
 |`cpus` | double | number of requested cpus.
 |`mem` | double | MB of requested memory.
+|`gpus` | double | Number of requested GPUs. Must be a whole number and support must be enabled in the config.
```

Comment:
We can't allocate fractional gpus _yet_, but that's on the roadmap eventually. So, it's a double.